ddtrace/tracer.py

```diff
@@ -91,6 +97,28 @@
 AnyCallable = TypeVar("AnyCallable", bound=Callable)
 
 
+def _start_appsec_processor():
+    # type: () -> Optional[Any]
+    # FIXME: type should be AppsecSpanProcessor but we have a cyclic import here
```

Comment:
you might be able to do `'AppsecSpanProcessor'` (a string) and have it work... but I am not sure since it is imported inside the function 🤔

Reply:
Yep, even with a string it fails without having the import before (which would case the cycle).